.github/workflows/backend-ci-cd.yml

```
@@ -0,0 +1,79 @@
+          ./setup-test-env.sh
+      - name: Start Infrastructure for Integration Tests
+        run: |
+          docker compose -f docker-compose.test.yaml --profile infra up -d
+          for i in {1..30}; do
+          if docker compose -f docker-compose.test.yaml --profile infra ps | grep "healthy"; then
+           echo "Infrastructure is healthy"
+           break
+          fi  
+          echo "Waiting for infrastructure to be healthy...($i/30)"
+          sleep 10
+          done
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)